src/bp/core/api.ts

```diff
@@ -40,6 +41,12 @@ const http = (httpServer: HTTPServer): typeof sdk.http => {
     },
     async getAxiosConfigForBot(botId: string, options?: sdk.AxiosOptions): Promise<any> {
       return httpServer.getAxiosConfigForBot(botId, options)
+    },
+    extractExternalToken(req: Request, res: Response, next: NextFunction): Promise<void> {
```

Comment:
does it makes sens to add this in the core ? will that be used somewhere else than the webchat ? You might want to move this to webchat backend

Reply:
Oh I just saw you added it in converse api as well

Reply:
I'm still not sure about the naming, there should be something we can add that indicates it's for external conversation auth and not external auth to botpress server itself

Reply:
lets get inspired by Java `extractConversationExternalAuthToken`